spec/system/conferences_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Visit conferences", type: :system do
+  let!(:organization) { create :organization }
+  let!(:conference_city) { create :conference, :published, organization: organization, hashtag: hashtag_city }
+  let!(:conference_global) { create :conference, :published, organization: organization, hashtag: hashtag_global }
+  let!(:conference_parallel) { create :conference, :published, organization: organization, hashtag: hashtag_parallel }
```

Comment:
```suggestion
  let!(:conference_city) { create :conference, :published, :promoted, organization: organization, hashtag: hashtag_city }
  let!(:conference_global) { create :conference, :published, :promoted, organization: organization, hashtag: hashtag_global }
  let!(:conference_parallel) { create :conference, :published, :promoted, organization: organization, hashtag: hashtag_parallel }
```

Let's use promoted conferences and handle the highlighted conferences block (it needs to respect the hashtags too)